### PR TITLE
Fix issue for extra fields not appearing in the errors

### PR DESCRIPTION
**Pull request by nofalx**
Solves Issue #835

Reference for impact of this change:
https://docs.pydantic.dev/latest/usage/validators/#model-validators

### CODE REVIEW

**Comment by nofalx** [comment]
This PR actually fixes another issue as well:

when u have Schema as attribute for another schema:
```
class UserTokenType(Schema):
    meta_data: MetaDataSchema | None = None
```
and you send this json
```
{
    "meta_data": "xxx",
}
```
Django ninja reports that schema is correct, however after the change in this pr we get the correct error that we missed before
```
{
    "error": "validation_errors",
    "details": [
        {
            "field_name": "meta_data",
            "msg": "Input should be a valid dictionary or object to extract fields from",
            "type": "model_attributes_type"
        }
    ]
```

**Comment by vitalik** [comment]
well it's not that easy (as you see from failed tests) 

I think there must be some trick so that pydantic thinks DjangoGetter is a dict

**Comment by vitalik** [comment]
Hi @nofalx 

regarding - `from_orm` ? did you find this somewhere on pydantic code ? I think they deprecated this keyword in favour to "from_attributes"...

**Comment by baseplate-admin** [comment]
Hi, Please use black on your code

**Comment by nofalx** [comment]
@vitalik  turns out the logic does not depend on the `from_attributes`  but more on if the source of validate if from from `model_validate` or somewhere else. 

I dont understand the inners of pydantic very well but I have tested the update code in a django projects and all scenarios seem to be working fine for root of the json and nested json shema we using for the API. The projects tests are all passing as well

**Comment by nofalx** [comment]
Added new test cases to cover the issues this pr fixed.  Code now passes lint and tests

File: ninja/signature/details.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def _create_models(self) -> TModels:
             )
 
             base_cls = param_cls._model
+            base_cls.model_config["from_attributes"] = True
```
Comment (vitalik) [inline ninja/signature/details.py:162]: what's the logic for "from_attributes" here ? the signature module kinda works only on anayling function signatures...

Comment (nofalx) [reply ninja/signature/details.py:162]: You are right, it was needed when I based my logic on `from_attributes` but its not valid anymore. Reverting

**Comment by vitalik** [comment]
There might be also development on pydantic side for this: https://github.com/pydantic/pydantic/issues/5434 will check if that covers this case as well

**Comment by vitalik** [comment]
Hi @nofalx 

I simplified a bit your approach (used class.model_config to find out if default validaion wrapper is needed) - see here for more details - #863 - let me know if that works ?

**Comment by nofalx** [comment]
Hi @vitalik 

Looks like a better approach, hope this makes it to next beta

**Pull request closed without merging**